### DISCUSSION TITLE
Unable to Create Argocd-Example guestbook on the local machine

### DISCUSSION THREAD

**Discussion by byteshiva** [argoproj/argo-cd#8632] [Q&A] [answered]
Unable to Create Argocd-Example guestbook on the local machine 

Error
```
Unable to create application: application spec for guestbook is invalid: 

InvalidSpecError: repository not accessible: rpc error: code = Unknown desc = error testing repository 
connectivity: Get "https://github.com/argoproj/argocd-example-apps.git/info/refs?service=git-upload-pack": dial 
tcp: lookup github.com on 10.96.0.10:53: server misbehaving
```

more errors
```
rpc error: code = Unknown 
desc = Get "https://github.com/argoproj/argocd-example-apps.git/info/refs?service=git-upload-pack": 
dial tcp: lookup github.com on 10.96.0.10:53: server misbehaving

```

**Comment by crenshaw-dev** [comment, marked as answer]
Can you confirm that you're able to clone that repo from your machine via a plain old `git clone`? It might just be an access issue.

**Reply by byteshiva** [reply]
I'm not able to git clone, As you mentioned, I assume it's a network access issue., I'm working on it for find a solution.

**Reply by byteshiva** [reply]
it works fine when I boot `minikube` with vm-driver as VirtualBox, ie.

```
minikube  start   --driver=virtualbox
```

but fails in case of docker as vm-driver.

**Reply by byteshiva** [reply]
As per documentation available at the below link, minikube with docker as vm-driver is not supported by argocd!

https://argo-cd.readthedocs.io/en/stable/developer-guide/toolchain-guide/
